decidim-core/app/uploaders/decidim/application_uploader.rb

```diff
@@ -75,11 +75,16 @@ def remote_url=(url)
     end
 
     def protocol_option
+      return { host: cdn_host } if cdn_host
```

Comment:
We don't need protocol here? The host configuration contains the protocol already?

Reply:
Yes I think so, I've been investigating and the host is processed with the protocol if exists, this allows for a single ENV var or this.
I think this is processed here:

https://github.com/rails/rails/blob/94b954576a0bb34ca81378b8a200d5d794a0a997/actionpack/lib/action_dispatch/http/url.rb#L114

if protocol is nil, it is extracted from the host. Am I right?